launcher/settingsView/csettingsview_moc.cpp

```diff
@@ -838,9 +838,17 @@ void CSettingsView::on_sliderScalingCursor_valueChanged(int value)
 
 void CSettingsView::on_buttonScalingAuto_toggled(bool checked)
 {
-	ui->spinBoxInterfaceScaling->setDisabled(checked);
-	ui->spinBoxInterfaceScaling->setValue(100);
-
+	if (checked)
+	{
+		ui->spinBoxInterfaceScaling->hide();
+	}
+	else
+	{
+		ui->spinBoxInterfaceScaling->show();
+		ui->spinBoxInterfaceScaling->setDisabled(false);
```

Comment:
if it's the only place where enabled/disabled state is changed, then this line isn't needed

Reply:
Good point. I'm on vacation now. I can update it afterwards.

Reply:
FYI you can edit the file directly on GitHub in your fork/branch

Reply:
```suggestion
```